core-ui/src/shared/components/ResourceDetails/ResourceDetails.js

```diff
@@ -183,12 +184,17 @@ function Resource({
   const editAction = () => {
     if (protectedResource) {
       return (
-        <Button
-          className="fd-margin-end--tiny"
-          onClick={() => openYaml(resource)}
+        <Tooltip
```

Comment:
Maybe we could add the same information in the editor, somewhere next to the `read only` information?
<img width="475" alt="Screen Shot 2022-07-28 at 14 22 32" src="https://user-images.githubusercontent.com/40663133/181503879-5522a84d-2060-4d67-8f80-20483a714de4.png">
(btw, the text is barely readable, if we agree to change the text, please also change its color)